.gitignore

```diff
@@ -10,3 +10,5 @@ tmp
 .classpath
 .project
 .settings
+*.swp
+tags
```

Comment:
vim creates a .swp file when a file is opened for editing in vim. The .swp file saves incremental changes (every 200 characters or 4 idle seconds) so that in the event of a system crash, say, you can recover your unsaved changes to a document open in vim.

I'm happy to put the tags ignoring (and the .swp ignoring) locally in `~/.config/git/ignore` instead of here. They are both user-practice specific, but then so is .DS_Store (Mac users) and .classpath (Eclipse users, I think?). So it really just comes down to where you draw the line.

In any case, I had not heard of the `~/.config/git/ignore` path before, and it's extremely useful! I can definitely move tags there, and if you'd prefer .swp files there too, can do.